common/Environments/Helpers/EnvironmentsNotificationHelper.cs

```diff
@@ -155,6 +160,11 @@ public void DisplayComputeSystemOperationError(string providerDisplayName, strin
     [RelayCommand]
     private void RestartComputer()
     {
+        TelemetryFactory.Get<ITelemetry>().Log(
+           "Environment_RestartNotification_ButtonClick_Event",
```

Comment:
I'd prefer to keep the naming convention in 3 sections, not 4.  Maybe something like "Environment_RestartComputer_Event"?

Reply:
sure will update